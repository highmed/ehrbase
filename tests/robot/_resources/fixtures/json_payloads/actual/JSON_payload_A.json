{
  "q":"X SELECT e/ehr_id/value FROM EHR e",
  "foo":"expected foo",
  "name": 1337,
  "columns":[1],
  "rows":{}
}
