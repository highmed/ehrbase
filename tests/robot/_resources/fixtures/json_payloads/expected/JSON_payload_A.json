{
  "TODO": "replace with expected result data-set"
}
